net-im/dino/dino-9999.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 2020 Gentoo Authors
```

Comment:
Change from "A-B" to "B" seemed a bit too sudden for me and I changed it manually to be still "A-B" (B being 2020 now). Maybe I missed an announcement of a policy change, in which case I'm sorry. But if that was your manual change, I made it to be the regular way.